docs/usage.md

```diff
@@ -96,6 +96,12 @@ Notably, this enables filtering for variants with certain impact levels or conse
 > [!NOTE]
 > The filtering step only works with conda for nextflow versions above 24.10.2 (use docker or singularity if you want to use an older nextflow version)
 
+### Subsample VCFs
```

Comment:
If you change the heading in `docs/output.md` then please change it here as well.